pkgs/top-level/all-packages.nix

```diff
@@ -11022,8 +11022,13 @@ let
 
   systemd = callPackage ../os-specific/linux/systemd {
     linuxHeaders = linuxHeaders_3_18;
+    cryptsetup = null; # Infinite recusion
   };
 
+  systemd-with-cryptsetup = appendToName "-with-cryptsetup" (systemd.override {
```

Comment:
This spelling seems inconsistent with `systemd_with_lvm2` which is defined right below it.